lib/winston/logger.js

```diff
@@ -42,30 +42,11 @@ class Logger extends Transform {
     this.configure(options);
   }
 
-  child(defaultRequestMetadata) {
+  child(childMetadata) {
     const logger = this;
     return Object.create(logger, {
-      write: {
```

Comment:
What are the full implications of taking out the `write` method?

Reply:
Will refresh my memory as to the purpose of this change. It's been a hot minute but I believe it was no longer used.

Reply:
I don't believe there are any implications here. The original implementation for the metadata of a child relied on an overridden write implementation that is seen here. By instead overriding the generation of the defaultMetadata for the child one can reuse the original write implementation without issue. As such I feel like this is a more appropriate implementation.